test/e2e/mxGraph.model.test.ts

```diff
@@ -931,10 +999,12 @@ describe('mxGraph model', () => {
     // boundary event: interrupting
     expect('boundary_event_interrupting_message_id').not.toBeCell();
     expect('boundary_event_interrupting_timer_id').not.toBeCell();
+    expect('boundary_event_interrupting_conditional_id').not.toBeCell();
 
     // boundary event: non-interrupting
     expect('boundary_event_non_interrupting_message_id').not.toBeCell();
     expect('boundary_event_non_interrupting_timer_id').not.toBeCell();
+    expect('boundary_event_non_interrupting_conditional_id').not.toBeCell();
```

Comment:
The addition of this event is missing in the **test/fixtures/bpmn/model-badly-attached-elements.bpmn** file 😅